rostime/include/ros/impl/time.h

```diff
@@ -167,7 +167,7 @@ namespace ros
 #if defined(BOOST_DATE_TIME_HAS_NANOSECONDS)
     return pt::from_time_t(sec) + pt::nanoseconds(nsec);
 #else
-    return pt::from_time_t(sec) + pt::microseconds(nsec/1000.0);
+    return pt::from_time_t(sec) + pt::microseconds(nsec/1000);
```

Comment:
> I have therefore made a small patch here, but also submitted a pull request to ros/roscpp_core#79. It might take quite some time to get this sorted and it might not even be a priority for ROS maintainers to consider, though...

👍 Looks like a real small issue in ros package indeed.